packages/node_modules/@ciscospark/react-component-activity-item-base/src/index.js

```diff
@@ -74,17 +77,20 @@ function ActivityItemBase({
     onActivityRetry(id);
   }
 
-  function getActionClassNames(highlight) {
+  function getActionClassNames(highlight, actionPending) {
```

Comment:
Can we start to use the object destructuring pattern the sdk uses for multiple options for functions? It makes it much easier to expand methods later on so we don't have to worry about order of passed params. (i.e. `getActionClassNames({highlight, actionPending})`). Is this something that makes sense, and we should add to the styleguide?

Reply:
I like it and I think it is in the style guide actually. It is just taking a while for it to sink in and become natural for me.